cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -0,0 +1,92 @@
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1),
+    pytest.mark.provider([ContainersProvider], scope='function')
+]
+
+TESTED_OBJECTS = [Container, Image, Service]
+PROJECT_NAME = 'test-project-dashboard'  # Predefine project for this test
+
+
+def get_api_object_counts(appliance, project_name, provider):
+    """ Fetches amount of Containers/Services/Images from the API per selected project name"""
+    assert isinstance(provider, ContainersProvider)
+    return {
+        Container: len(provider.mgmt.list_container(project_name=project_name)),
+        Service: len(provider.mgmt.list_service(project_name=project_name)),
+        Image: len(get_container_images_amt(provider, project_name))
+    }
+
+
+def get_container_images_amt(provider, project_name=None):
+    """ Fetches images amount from the API per selected project name"""
+    images = provider.mgmt.list_image()
+    project_images = []
+    for image in images:
+        if image.image_project_name == project_name:
+            project_images.append(image)
+    return project_images
+
+
+def get_api_pods_names(provider):
+    """ Fetches Pod names from the API per selected project name"""
+    pod_name = []
+    for pod in provider.mgmt.list_container_group(project_name=PROJECT_NAME):
+        pod_name.append(pod.name)
+    return pod_name
+
+
+@pytest.mark.polarion('CMP-10806')
+def test_projects_dashboard_pods(provider, appliance, soft_assert):
+    """Tests data integrity of Pods names in Pods status box in Projects Dashboard.
+    Steps:
+        * Go to Projects / Dashboard View
+        * Compare the data in the Pods status box to API data for
+        Pods names
+    """
+    api_pod_names = get_api_pods_names(provider)
+    collection_object = appliance.collections.container_projects
+    inst = collection_object.instantiate(name=PROJECT_NAME, provider=provider)
+    view = navigate_to(inst, 'Dashboard')
+    for field in view.pods.fields:
+        soft_assert(
+            field in api_pod_names,
+            'There is a mismatch between API and UI values: {} (API) != {} (UI)'.format(
+                api_pod_names, field
+            )
+        )
+
+
+@pytest.mark.polarion('CMP-10805')
+def test_projects_dashboard_icons(provider, appliance, soft_assert):
+    """Tests data integrity of Containers/Images/Services number in
+    Projects Dashboard's status boxes.
+    Steps:
+        * Go to Projects / Dashboard View
+        * Compare the data in the status boxes to API data forz
+        Containers/Images/Services numbers
+    """
+    api_values = get_api_object_counts(appliance, PROJECT_NAME, provider)
+    collection_object = appliance.collections.container_projects
+    inst = collection_object.instantiate(name=PROJECT_NAME, provider=provider)
+    view = navigate_to(inst, 'Dashboard')
+    for obj in TESTED_OBJECTS:
+        statusbox_value = getattr(view, obj.PLURAL.split(' ')[-1].lower()).value
```

Comment:
document in-line what you're trying to get from `obj.PLURAL.split()[-1].lower.value, as its not at all clear what's being parsed.